meson.build

```diff
@@ -0,0 +1,31 @@
+project('nwg-launchers', 'cpp',
+	default_options: ['cpp_std=c++17'],
```

Comment:
You might want to set a warning level as well, something like `'warning_level=2'` or even `3`.